server/routes/comments.js

```diff
@@ -1,80 +1,95 @@
 var express = require ('express')
 var router = express.Router()
-const db = require ('../database/db.js')
+const comment = require('../database/queries/comment')
+
 
 //get all comments
 router.get('/', async (req, res) => {
     try {
-        let allComments = await db.any('SELECT * FROM comments;')
+        let allComments = await comment.getAllComments()
         res.json({
-            body: allComments
+            payload: allComments,
+            msg: 'success',
+            err: false
         })
     } catch (error) {
-        res.json({
-            error: error
+        console.log(error)
+        res.status(500).json({
+            payload: null,
+            msg: error,
```

Comment:
outputting errors to the frontend can be a security issue because it gives away information about your system. It can be useful for debugging however. Only output error information in response if you can control what the error is is my recommendation.